fix: Add connecting services to node + express (#3418)
getsentry/sentry-docs: src/platforms/node/common/performance/index.mdx

  res.status(200).end();
});
```

## Connecting Services

> If you are also using Performance Monitoring for [JavaScript](/platforms/javascript/performance/) depending on where your request originates there are two methods to connect your traces:
> 1. For requests that start in your backend, this is done by [adding a meta tag](/platforms/javascript/performance/connect-services/#pageload) in your HTML template that contains tracing information.
> 2. For requests that start in JavaScript, this is done by the SDK [setting a header](/platforms/javascript/performance/connect-services/#navigation-and-other-xhr-requests) on requests to your backend.
> 
> Otherwise backend services with Performance Monitoring will connect automatically.

Reviewer: ```suggestion
If you are also using Performance Monitoring for [JavaScript](/platforms/javascript/performance/), depending on where your request originates, you can connect traces:
1. For requests that start in your backend by [adding a meta tag](/platforms/javascript/performance/connect-services/#pageload) in your HTML template that contains tracing information.
2. For requests that start in JavaScript by the SDK [setting a header](/platforms/javascript/performance/connect-services/#navigation-and-other-xhr-requests) on requests to your backend.

Otherwise, backend services with Performance Monitoring connect automatically.
```
